airflow/config_templates/config.yml.schema.json

```diff
@@ -72,6 +72,11 @@
                 "sensitive": {
                     "type": "boolean",
                     "description": "When true, this option is sensitive and can be specified using AIRFLOW__{section}___{name}__SECRET or AIRFLOW__{section}___{name}_CMD environment variables. See: airflow.configuration.AirflowConfigParser.sensitive_config_values"
+                },
+                "is_template": {
+                    "type": "boolean",
+                    "description": "When true, this option is not processed by format after being processed by AirflowConfigParser, The JINJA templates should be encoded as {{{{ }}}} (they are converted to {{ }} in the final field default), whereas the Python format strings should be encoded as `{{ }}` and they are converted to { } in the final field default)",
+                    "default": false
```

Comment:
```suggestion
                    "description": "When true, this option is not processed by formatter after being processed by AirflowConfigParser, The JINJA templates should be encoded as {{{{ }}}} (they are converted to {{ }} in the final field default), whereas the Python format strings should be encoded as `{{ }}` and they are converted to { } in the final fields)",
                    "default": false
```

Reply:
Actually this description is slightly wrong (I managed to change it in the way that we do not need `{{{{` any more. I will update the description to be way shorter as well.